.gitmodules

```diff
@@ -1,3 +1,4 @@
-[submodule "workflow/proc_pipe/tractoflow/tractoflow"]
-	path = workflow/proc_pipe/tractoflow/tractoflow
-	url = git@github.com:scilus/tractoflow.git
+[submodule "nipoppy/workflow/proc_pipe/tractoflow/tractoflow"]
+	path = nipoppy/workflow/proc_pipe/tractoflow/tractoflow
+	url = https://github.com/scilus/tractoflow.git
+	datalad-url = https://github.com/scilus/tractoflow.git
```

Comment:
`main.nf` w/in the tractoflow repo is the analysis script that nextflow invokes when it runs. Nextflow is basically a generalized nipype w/ it's own syntax to wrap the commands. I have to point at `main.nf` and the container for this version - the newest release has the `*.nf` files w/in the container. This is the version running on UKBB, so preserving for posterity is desirable. Let me know if there's a more convenient way to package this together.

Reply:
best to be sure that it will keep on working is to... write a test for it... 😉 

let me check but I am sure we can package git submodule into python packages #inception

Reply:
@bcmcpher https://github.com/neurodatascience/nipoppy/pull/132/commits/f828e0118280e4d7f72965c0f2a7df12987e1dbc should fix it